### DISCUSSION TITLE
Extending Trackball Controls

### DISCUSSION THREAD

**Discussion by nart4hire** [Tresjs/tres#755] [Q&A]
Hi, I've been experimenting with Tres for a bit and came across a particular use case that requires trackball controls (gotta love spherical objects), however extending it according to the documentation does not seem to work.

Here's a stackblitz of what I've been trying to do: [Stackblitz](https://stackblitz.com/edit/tresjs-minimal-reproduction-vev5jt?file=src%2Fcomponents%2FTheExperience.vue)

As you can see, if we uncomment the orbit controls and use that instead of trackball controls, the experience works. I'm not sure where I'm going wrong. Any help would be appreciated!

**Comment by andretchen0** [comment]
Hey @nart4hire ,

Thanks for bringing this up. @JaimeTorrealba had a look at it and confirms that this is a bug in Cientos.

Would you please create a [bug report](https://github.com/Tresjs/cientos/issues/new?assignees=&labels=pending+triage&projects=&template=bug.report.yml) and include your StackBlitz? Thanks!
